test/language/computed-property-names/computed-property-names-prototype.js

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) Copyright 2014 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 12.2.5
+description: >
+    computed property names can be "prototype"
+---*/
+class C {
+  static ['prototype']() {
```

Comment:
This might be correct (???) --- The error is specified [here](https://people.mozilla.org/~jorendorff/es6-draft.html#sec-class-definitions-static-semantics-early-errors), but for a computed property name you can't really have an early error like that. The spec doesn't say anything about it being a runtime error, afaik.